packages/ens-controller/src/EnsController.ts

```diff
@@ -6,7 +6,12 @@ import {
   normalizeEnsName,
   isValidHexAddress,
   toChecksumHexAddress,
+  NetworksChainId,
+  CHAIN_ID_TO_NETWORK_ID_MAP,
 } from '@metamask/controller-utils';
+import { toASCII } from 'punycode/';
```

Comment:
I think it's due to what they are saying here https://www.npmjs.com/package/punycode
Note that userland modules don't hide core modules. For example, require('punycode') still imports the deprecated core module even if you executed npm install punycode. Use require('punycode/') to import userland modules rather than core modules.

Reply:
Oh... wow... that's very interesting. They really should have just renamed this package I think instead of doing this. Oh well. Okay thanks for looking into this.